### DISCUSSION TITLE
How to disable Persistent Comment Line Insertion in Neovim Upon Pressing Enter

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#2184] [Q&A] [answered]
I'm encountering an unusual issue in Neovim where pressing Enter while in a commented line automatically inserts a new commented line below. This behaviour persists even after disabling both `mini.comment` and other comment plugins. The issue seems unrelated to these commenting plugins, as it occurs even when they are inactive. I've checked my configuration and auto-commands but haven't identified a cause. 

https://github.com/LazyVim/LazyVim/assets/51330172/767a972c-e4ab-4369-8d8f-ebff6def0936

Does Lazyvim have any settings that cause this behaviour?

**Comment by dpetka2001** [comment, marked as answer]
See `:h 'formatoptions'` and `:h fo-table`. What you want is the `r` flag. Set it as an auto-command because there exist plugins that may override it. Have a look at how I do it in my [dotfiles](https://github.com/dpetka2001/dotfiles/blob/43ecf1a6c1f1c8bf4ffe6fbc591f46db25b592db/dot_config/nvim/lua/config/autocmds.lua#L31-L44) and make appropriate changes to your needs.
